### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ArtPoon**
* #28 
* #442 
* #459 
* #464 
* #468 
* #447 
* #453

### CODE REVIEW

File: covizu/utils/batch_utils.py [hunk 1]
```diff
@@ -98,7 +95,7 @@ def beadplot_serial(lineage, features, args, callback=None):
         beaddict.update({'sampled_variants': len(labels)})
         beaddict['nodes'].update({variant: []})
 
-        for coldate, accn, label1 in intermed:
+        for coldate, accn, location, label1 in intermed:
```
Comment (ArtPoon) [inline covizu/utils/batch_utils.py:98]: this is a deprecated function - this change was made to allow a unit test to pass

**Comment by GopiGugan** [comment]
* Checkout `dev` branch
* `python3 setup.py install --user`
* `module load openmpi/gnu`
* `nohup python3 batch.py --dry-run > pr473.log &`

**Comment by ArtPoon** [comment]
testing in progress

**Comment by GopiGugan** [comment]
Recombinants aren't being displayed:

image

Related to change made in 4f74df1286699b4934fc1f4ccfb24c26bdfdcbc7

**Comment by GopiGugan** [comment]
Checking the following:
- [ ] Search
    - [ ] Search by accession number
    - [ ] Search by lineage
    - [ ] Search by country
    - [ ] Search by country (with date filter)
- [ ] Tree filtering
- [ ] Download Time-scaled tree (nwk)
- [ ] Download Time-scaled tree (csv)
- [ ] Download Beadplot (svg)
- [ ] Download Beadplot (nwk)
- [ ] Download mutation list (csv)
- [ ] Beadplot cutoff slider
- [ ] Expanding Beadplot
- [ ] Translated pages
- [ ] Changing the colour of the tree based on divergence, collection date, etc.
- [ ] Left/Right arrow key to move between beads
- [ ] Tooltips
- [ ] Countries/Samples/Mutations tabs are populating the correct information
- [ ] Toggle for displaying recombinants

**Comment by GopiGugan** [comment]
Unit tests are failing:

```
  11 passing (1m)
  3 failing

  1) Search Interface
       Searching 'China' results in selection and expected point count:
     AssertionError: Timed out retrying after 10000ms: Expected to find element: `.selectionH`, but never found it.
      at Context.eval (webpack:///./cypress/e2e/covizu.spec.cy.js:93:30)

  2) Search Interface
       Clicking the next button highlights the correct bead:
     TypeError: Cannot read properties of undefined (reading '0')
      at Context.eval (webpack:///./cypress/e2e/covizu.spec.cy.js:101:90)

  3) Search Interface
       Clear button resets search interface and cluster selection:
     CypressError: Timed out retrying after 10050ms: `cy.click()` failed because this element is `disabled`:
```

**Comment by GopiGugan** [comment]
Unit test did not timeout when re-running the tests

**Comment by GopiGugan** [comment]
```js
Uncaught ReferenceError: Driver is not defined
    at HTMLDivElement.click (covizu.js:46:22)
    at HTMLButtonElement.<anonymous> (jquery-ui.min.js:11:4603)
    at HTMLButtonElement.dispatch (jquery.js:5429:27)
    at elemData.handle (jquery.js:5233:28)
```

To reproduce:
1. Click `es` to navigate to the Spanish translated page
2. Click on Tour rapido

**Comment by GopiGugan** [comment]
```js
Uncaught DOMException: Failed to execute 'querySelector' on 'Document': '.lines#India-UT-VCSGGIMS&R-VCSG416285' is not a valid selector.
    at Object.select (http://localhost:8001/js/d3.js:1689:34)
    at SVGLineElement.<anonymous> (http://localhost:8001/js/beadplot.js:752:30)
    at SVGLineElement.<anonymous> (http://localhost:8001/js/d3.js:1526:16)
```

To reproduce:
1. Open the `BA.2.75.5` beadplot
2. Hover over the vertical line connecting the `India/UT-VCSGGIMS&R-VCSG416274` and `India/UT-VCSGGIMS&R-VCSG416285` beads

**Comment by ArtPoon** [comment]
Can we deactivate the tests that are affected by the sporadic timeout issue, and run them manually with each PR?

**Pull request merged**